Realm/RLMObjectSchema.mm

```diff
@@ -105,7 +91,7 @@ + (instancetype)schemaForObjectClass:(Class)objectClass {
     Class cls = objectClass;
     Class superClass = class_getSuperclass(cls);
     NSArray *props = @[];
-    while (superClass != RLMObjectBase.class) {
+    while (superClass && superClass != RLMObjectBase.class) {
```

Comment:
Why is this change needed? It only behaves differently if `RLMObjectBase.class` is nil.